tests/test_gso.cpp

```diff
@@ -210,7 +286,7 @@ template <class ZT, class FT> int test_int_rel(int d, int b)
         << " shows different GSO-outputs for grammatrix representation and basis representation.\n";
     return 1;
   }
-  return 0;
+  return test_householder<ZT, FT>(A);
```

Comment:
Maybe do this call in the same style as the above calls to keep the symmetry?

Reply:
This must be done in commit 41fb8501dcc70bbc8511ba10971a70b69d0fc5e4.